packages/gasket-plugin-docusaurus/lib/docs-view.js

```diff
@@ -0,0 +1,28 @@
+const defaultsDeep = require('lodash.defaultsdeep');
+const { existsSync } = require('fs');
+const { writeFile } = require('fs').promises;
+const { start } = require('@docusaurus/core/lib');
+const path = require('path');
+const generateDefaultConfig = require('./generate-default-config');
+const pluginConfigFile = 'docusaurus.config.js';
+const defaultConfig = {
+  port: 3000,
+  host: 'localhost'
+};
+
+module.exports = async function docsView(gasket, docsConfigSet) {
+  const { docsRoot } = docsConfigSet;
```

Comment:
Move the require statement to within the function handler to avoid loading this module when it is not needed.
```suggestion
  const { start } = require('@docusaurus/core/lib');
  const { docsRoot } = docsConfigSet;
```